File: Fundamentals/Functions - Exercise/perfect_number.py
def is_perfect_number(number: int):
    sum_of_index = 0
    for i in range(1, number+1):
        if number % i == 0:
            sum_of_index += i
    if sum_of_index / 2 == number:
        return 'We have a perfect number!'
    else:
        return "It's not so perfect."


current_number = int(input())
print(is_perfect_number(number=current_number))

File: JS Front End/Functions and Statements - Exercises/perfectNumber.js
function isPerfectNumber(numb) {
    let result = [];
    for (let i = 1; i < numb; i++) {
        if (numb % i === 0) {
            result.push(i);
        }
    }
    let divisors = eval(result.join('+'));
    divisors === numb
        ? console.log('We have a perfect number!')
        : console.log('It\'s not so perfect.');
}

isPerfectNumber(2);